src/pages/Charcuterie.js

```diff
@@ -77,7 +77,7 @@ export default () => {
         <br></br>
         <QuestionHeading>this is a question heading</QuestionHeading>
         <QuestionHeading>question 14</QuestionHeading>
-        <H1>How did you hear about nwHacks?</H1>
+        <H1 size="1.5em">How did you hear about nwHacks?</H1>
```

Comment:
if we need a larger H1 for application questions, we can just create a styled H1 like so

```jsx
const QuestionSubheading = styled(H1)`
	size: 1.5em;
`
```

Reply:
I initially made this exact component but then noticed H1 had a size conditional so I used that instead, can bring the styled H1 component back though :))